src/ImageSharp/ImageSharp.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageTags>Image Resize Crop Gif Jpg Jpeg Bitmap Png Tga NetCore</PackageTags>
     <Description>A new, fully featured, fully managed, cross-platform, 2D graphics API for .NET</Description>
     <Configurations>Debug;Release;Debug-InnerLoop;Release-InnerLoop</Configurations>
+    <DefineConstants Condition="'$(Configuration)' == 'Debug-InnerLoop'">$(DefineConstants);DEBUG</DefineConstants>
```

Comment:
I'm surprised this is required, however I had to do something similar to ensure release builds are optimized. You'll want to move it up to the root `Directory.Build.props` to live with that command.

Reply:
Yeah, I noticed this by realizing that `DebugGuard` doesn't work in `Debug-Innerloop`. Will promote to `Directory.Builld.props`, but might be something to look into at SharedInfrastructure level even.